aapcs64/aapcs64.rst

```diff
@@ -2026,6 +2026,21 @@ size\_t, ptrdiff\_t
 
 See `Arm C and C++ language mappings`_.
 
+Soft-float variant
+------------------
+
+For v8-R CPUs which do not have floating-point instructions or registers, the soft-float variant of the PCS can be used. Arm recommends that this is not used more widely, as most existing AArch64 implementations do have floating-point registers, and this variant is incompatible with the base variant.
```

Comment:
I think we could use stronger language here. Perhaps
The soft-float variant is defined for Arm v8-R implementations that do not have floating-point instructions or registers. This variant is incompatible with the base procedure call standard and toolchains are not required to support it.

Reply:
Done